test/uk/gov/hmrc/gform/gform/StructuredFormDataBuilderSpec.scala

```diff
@@ -50,9 +48,7 @@ class StructuredFormDataBuilderSpec extends Spec {
       createForm(
         "field" -> "value1,value2"
       ),
-      objectStructure(
-        "field" -> arrayNode(textNode("value1"), textNode("value2"))
-      )
+      objectStructure(field("field", arrayNode(textNode("value1"), textNode("value2")), Map.empty))
```

Comment:
Make our lives easy & remove the duplication by giving the alternateFieldNames parameter of field(...) a default, Map.empty.